### ISSUE TITLE
startup is 4x as expensive in the packaged version

### ISSUE THREAD

**Issue by oconnor663** [buildinspace/peru#178]
In my local repo:

```
$ pacman -Qo /usr/bin/peru
/usr/bin/peru is owned by peru-git 605.1610b46-1
$ time /usr/bin/peru > /dev/null
/usr/bin/peru > /dev/null  0.41s user 0.02s system 99% cpu 0.429 total
$ time ./peru.py > /dev/null
./peru.py > /dev/null  0.11s user 0.00s system 99% cpu 0.115 total
```

**Comment by olson-sean-k** [comment]
Interesting. Confirmed on my machine as well. Packaged version consistently clocked in at ~0.38s while running directly from a clone clocked in at ~0.11s.

**Comment by olson-sean-k** [comment]
Possibly related: https://github.com/pypa/setuptools/issues/510

**Comment by oconnor663** [comment]
Nice! I bet that's it, though I haven't played with it yet.

**Comment by oconnor663** [comment]
Maybe using "scripts" instead of "entry_points" in setup.py will just work around this? I think that's what we used to do anyway.

**Comment by oconnor663** [comment]
This should be fixed with ba92556d0e4881008720cc3c61f1cefc6d383723, which vendors in a copy of https://github.com/ninjaaron/fast-entry_points/blob/master/fastentrypoints.py.

**Comment by oconnor663** [comment]
Tested on both Linux and Windows with peru version 1.1.3. Seems to run twice as fast as 1.1.2, and equally as fast as `python -m peru`. (Not sure what happened to the 4x multiple.)
